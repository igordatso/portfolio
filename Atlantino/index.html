<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>ATLANTINO</title>
    <link rel="stylesheet" href="css/normalize.min.css">
    <link rel="stylesheet" type="text/css" href="css/slick.css"/>
    <link rel="stylesheet" type="text/css" href="css/slick-theme.css"/>
    <link rel="stylesheet" href="css/style.css">
</head>
<body class="home">
<nav id="sideNav">
    <ul class="list-wrap">
        <li class="active"><a href="#home">Home</a></li>
        <li><a href="#bio">Bio</a></li>
        <li><a href="#music">Music</a></li>
        <li><a href="#video">Videos</a></li>
        <li><a href="#">Store</a></li>
        <li><a href="#">Contact</a></li>
        <li><a href="#">Donate</a></li>
    </ul>
</nav>
<header class="clearfix">
    <span class="nav-button open-side-nav"></span>
    <div class="social-music-block">
        <span class="buy-music-button">buy music / stores <i class="music-icon"></i></span>
        <ul class="music-social-icons">
            <li><a href="#"><span class="social-title">dsfsdfsdfdfs</span><i class="one-icon spotify"></i></a></li>
            <li><a href="#"><span class="social-title">Spotify/Atlantino</span><i class="one-icon spotify"></i></a></li>
            <li><a href="#"><span class="social-title">dsfsdfsdfdfs</span><i class="one-icon soundcloud"></i></a></li>
            <li><a href="#"><span class="social-title">Spotify/Atlantino</span><i class="one-icon spotify"></i></a></li>
        </ul>
    </div>
</header>
<div class="social-icons">
    <a href="#"><i class="facebook"></i></a>
    <a href="#"><i class="twitter"></i></a>
    <a href="#"><i class="instagram"></i></a>
    <a href="#"><i class="youtube"></i></a>
</div>

<div id="home" class="partial">
    <div class="logo"></div>
    <div class="atlantino"></div>
</div>

<div id="bio" class="partial">
    <div class="bio-text">
        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum luctus ut neque in volutpat. Mauris sollicitudin neque bibendum fringilla porta. Nulla hendrerit tortor ac arcu interdum finibus. Maecenas at arcu lobortis, tincidunt mi ac, hendrerit magna. Nam aliquet sit amet ligula id molestie. Praesent non lorem ut quam auctor vestibulum vel sed nisi. Vivamus a diam ante. Sed sit amet feugiat metus, nec vehicula neque.</p>
        <p>Nulla velit nunc, sodales sed odio vel, cursus luctus erat. Ut mollis mi in metus gravida congue. Nulla vel felis cursus, cursus massa ut, vehicula purus. Aliquam erat volutpat. Nullam tortor ante, pharetra nec malesuada ac, pretium a justo. Sed lacinia eu tellus vel porttitor. Vivamus cursus enim libero, et facilisis tellus porttitor a. Aliquam euismod, augue et commodo efficitur, ligula lacus consectetur sem, sed laoreet dui elit sit amet elit. Quisque vel ante eu ex euismod sagittis. Phasellus ac justo urna. Suspendisse aliquet sem eget dui accumsan, et consequat sem consequat. Morbi nec vehicula eros. Ut fermentum justo vitae lectus eleifend placerat. Vestibulum ut sem consequat nunc.</p>
    </div>
</div>

<div id="music" class="partial">
    <div class="slider">
        <div class="vertical-slider">
            <div><a href="#"><img src="img/music-slider-big.jpg" alt=""></a></div>
            <div><a href="#"><img src="img/slider-social-icons.jpg" alt=""></a></div>
            <div><a href="#"><img src="img/music-slider-big.jpg" alt=""></a></div>
            <div><a href="#"><img src="img/slider-social-icons.jpg" alt=""></a></div>
            <div><a href="#"><img src="img/music-slider-big.jpg" alt=""></a></div>
            <div><a href="#"><img src="img/slider-social-icons.jpg" alt=""></a></div>
            <div><a href="#"><img src="img/music-slider-big.jpg" alt=""></a></div>
            <div><a href="#"><img src="img/slider-social-icons.jpg" alt=""></a></div>
        </div>
    </div>
</div>

<div id="video" class="partial">
    <div class="video-block">
        <img src="img/video-block.jpg" alt="">
    </div>
</div>

<div class="cssload-container">
    <div class="cssload-whirlpool"></div>
</div>

<script src="js/jquery-3.3.1.min.js"></script>
<script src="js/slick.js"></script>
<script src="js/init.js"></script>
</body>
</html>