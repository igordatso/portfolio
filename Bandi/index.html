
<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="css/normalize.css"/>
<link rel="stylesheet" type="text/css" href="css/style.css"/>

	<title>Bandi</title>
</head>
<body>

	<header>
		<div class="nav-wrap clearfix">
			<div class="container">
				<div class="logo-section">
					<a class="logo" href="#">Brandi</a>
				</div>
				<div class="text-section">
						<span class="logo-text">I'am your tag line</span>
				</div>
				<nav class="nav">
					<ul>
						<li><a href="#">Home</a></li>
						<li><a href="#">Features</a></li>
						<li><a href="#">Works</a></li>
						<li><a href="#">Team</a></li>
						<li><a href="#">Contact</a></li>
					</ul>
				</nav>
			</div>
		</div>
		<div class="header-slider-item clearfix">
			<img src="images/ellipse1.png" alt="ellipse"/><br>
			<img src="images/ellipse2.png" alt="ellipse"/><br>
			<img src="images/ellipse2.png" alt="ellipse"/>
		</div>
		<div class="container clearfix">
			<div class="header-text">
				<h1>Meet <span>Brandi</span>!</h1>
				<h2><span>/Creative </span>one page template.</h2>
				<p><span>We are a team of professionals</span></p>
			</div>
			<div class="social">
				<a href="#"><img src="images/social_1.png" alt=""></a>
				<a href="#"><img src="images/social_2.png" alt=""></a>
				<a href="#"><img src="images/social_3.png" alt=""></a>
				<a href="#"><img src="images/social_4.png" alt=""></a>
			</div>
		</div>
	</header>

	<div class="features">
		<div class="container">
			<div class="heading-title">
				<h3>Features</h3>
				<span><img src="images/heart.png" alt="heart"/></span>
			</div>
			<div class="features-content clearfix">
				<div class="features-item">
					<div class="features-title">
							<h4 class="f-1">Branding</h4>
					</div>
					<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore</p>
				</div>
				<div class="features-item">
					<div class="features-title">
							<h4 class="f-2">Development</h4>
					</div>
					<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore</p>
				</div>
				<div class="features-item">
					<div class="features-title">
							<h4 class="f-3">Consulting</h4>
					</div>
					<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore</p>
				</div>
			</div>
			<div class="slider-item">
				<img src="images/ellipse3.png" alt="ellipse"/>
				<img src="images/ellipse4.png" alt="ellipse"/>
				<img src="images/ellipse4.png" alt="ellipse"/>
			</div>
		</div>
	</div>

	<div class="works">
		<div class="container">
			<div class="heading-title">
				<h3>Works</h3>
				<span><img src="images/heart.png" alt="heart"/></span>
			</div>
			<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam <br/> rem aperiam, eaque ipsa quae ab illo inventore</p>
			<ul class="works-nav">
				<li><a href="#" class="works-active">All</a></li>
				<li><a href="#">Branding</a></li>
				<li><a href="#">Web</a></li>
				<li><a href="#">Logo Design</a></li>
				<li><a href="#">Photography</a></li>
			</ul>
		</div>
		<div class="work-items clearfix">
			<div class="work-item">
				<img src="images/works1.jpg" alt="work1">
				<div class="work-overlay">
					<img src="images/oko.png" alt="oko">
					<p>Labore et dolore magnam <span>Photography</span> </p>
				</div>
			</div>
			<div class="work-item">
				<img src="images/works2.jpg" alt="work2">
				<div class="work-overlay">
					<img src="images/oko.png" alt="oko">
					<p>Labore et dolore magnam <span>Photography</span> </p>
				</div>
			</div>
			<div class="work-item">
				<img src="images/works3.jpg" alt="work3">
				<div class="work-overlay">
					<img src="images/oko.png" alt="oko">
					<p>Labore et dolore magnam <span>Photography</span> </p>
				</div>
			</div>
			<div class="work-item">
				<img src="images/works4.jpg" alt="work4">
				<div class="work-overlay">
					<img src="images/oko.png" alt="oko">
					<p>Labore et dolore magnam <span>Photography</span> </p>
				</div>
			</div>
			<div class="work-item">
				<img src="images/works5.jpg" alt="work5">
				<div class="work-overlay">
					<img src="images/oko.png" alt="oko">
					<p>Labore et dolore magnam <span>Photography</span> </p>
				</div>
			</div>
			<div class="work-item">
				<img src="images/works6.jpg" alt="work6">
				<div class="work-overlay">
					<img src="images/oko.png" alt="oko">
					<p>Labore et dolore magnam <span>Photography</span> </p>
				</div>
			</div>
			<div class="work-item">
				<img src="images/works7.jpg" alt="work7">
				<div class="work-overlay">
					<img src="images/oko.png" alt="oko">
					<p>Labore et dolore magnam <span>Photography</span> </p>
				</div>
			</div>
			<div class="work-item">
				<img src="images/works8.jpg" alt="work8">
				<div class="work-overlay">
					<img src="images/oko.png" alt="oko">
					<p>Labore et dolore magnam <span>Photography</span> </p>
				</div>
			</div>
		</div>
	</div>

	<div class="meet_our_team">
		<div class="container">
			<div class="heading-title">
				<h3>Meet Our Team</h3>
				<span><img src="images/heart.png" alt="heart"/></span>
			</div>
			<p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam <br/> rem aperiam, eaque ipsa quae ab illo inventore</p>
			<div class="meet-items clearfix">
				<div class="meet-block">
					<div class="meet-item">
						<img src="images/team1.jpg" alt="meet4">
						<div class="meet-overlay">
							<div class="text">
								<p>Nemo enim ipsam voluptatem quia voluptas <span>sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.</span> </p>
							</div>
							<div class="social">
								<div class="ramka twiter">
									<a href="#"><img src="images/team1.png" alt="social"></a>
								</div>
								<div class="ramka">
									<a href="#"><img src="images/team2.png" alt="social"></a>
								</div>
								<div class="ramka">
									<a href="#"><img src="images/team3.png" alt="social"></a>
								</div>
							</div>
						</div>
					</div>
				<div class="meet-item-txt">
					John Filmr Doe <br>
					<span>Managing Director</span>
				</div>
			</div>

			<div class="meet-block">
				<div class="meet-item">
					<img src="images/team2.jpg" alt="meet4">
					<div class="meet-overlay">
						<div class="text">
							<p>Nemo enim ipsam voluptatem quia voluptas <span>sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.</span> </p>
						</div>
						<div class="social">
							<div class="ramka twiter">
								<a href="#"><img src="images/team1.png" alt="social"></a>
							</div>
							<div class="ramka">
								<a href="#"><img src="images/team2.png" alt="social"></a>
							</div>
							<div class="ramka">
								<a href="#"><img src="images/team3.png" alt="social"></a>
							</div>
						</div>
					</div>
				</div>
				<div class="meet-item-txt">
					John Filmr Doe<br>
					<span>Managing Director</span>
				</div>
			</div>

			<div class="meet-block">
				<div class="meet-item">
					<img src="images/team3.jpg" alt="meet4">
					<div class="meet-overlay">
						<div class="text">
							<p>Nemo enim ipsam voluptatem quia voluptas <span>sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.</span> </p>
						</div>
						<div class="social">
							<div class="ramka twiter">
								<a href="#"><img src="images/team1.png" alt="social"></a>
							</div>
							<div class="ramka">
								<a href="#"><img src="images/team2.png" alt="social"></a>
							</div>
							<div class="ramka">
								<a href="#"><img src="images/team3.png" alt="social"></a>
							</div>
						</div>
					</div>
				</div>
				<div class="meet-item-txt">
					John Filmr Doe<br>
					<span>Managing Director</span>
				</div>
			</div>

			<div class="meet-block">
				<div class="meet-item">
					<img src="images/team4.jpg" alt="meet4">
					<div class="meet-overlay">
						<div class="text">
							<p>Nemo enim ipsam voluptatem quia voluptas <span>sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.</span> </p>
						</div>
						<div class="social">
							<div class="ramka twiter">
								<a href="#"><img src="images/team1.png" alt="social"></a>
							</div>
							<div class="ramka">
								<a href="#"><img src="images/team2.png" alt="social"></a>
							</div>
							<div class="ramka">
								<a href="#"><img src="images/team3.png" alt="social"></a>
							</div>
						</div>
					</div>
				</div>
				<div class="meet-item-txt">
					John Filmr Doe<br>
					<span>Managing Director</span>
				</div>
			</div>
		</div>
			<div class="slider-item">
				<img src="images/ellipse3.png" alt="ellipse"/>
				<img src="images/ellipse4.png" alt="ellipse"/>
				<img src="images/ellipse4.png" alt="ellipse"/>
			</div>
		</div>
	</div>
	<div class="some_fun_facts">
		<div class="container">
			<div class="heading-title">
				<h3>Some Fun Facts</h3>
				<span><img src="images/heart2.png" alt="heart"/></span>
			</div>
			<div class="some-items clearfix">
				<div class="some-block">
					<div class="some-item">
						<img src="images/hours1.png" alt="meet4">
					</div>
					<div class="some-item-txt">3200 <br> <span>Hours of Work</span></div>
				</div>
				<div class="some-block">
					<div class="some-item">
						<img src="images/hours2.png" alt="meet4">
					</div>
					<div class="some-item-txt">120 <br> <span>Satisfied Clinets</span></div>
				</div>
				<div class="some-block">
					<div class="some-item">
						<img src="images/hours3.png" alt="meet4">
					</div>
					<div class="some-item-txt">360 <br> <span>Projects delivered</span></div>
				</div>
				<div class="some-block">
					<div class="some-item">
						<img src="images/hours4.png" alt="meet4">
					</div>
					<div class="some-item-txt">42 <br> <span>Awards Won</span></div>
				</div>
			</div>
		</div>
	</div>

	<div class="lets_discuss">
		<div class="container">
			<div class="heading-title">
				<h3>Let’s Discuss</h3>
				<span><img src="images/heart.png" alt="heart"/></span>
			</div>
			<p>Voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore</p>
			<div class="forma">
					<div class="forma-left">
						<p>Cras at ultrices erat, sed vulputate!</p>
						<span>2345 Setwant natrer, 1234,<br>Washington. United States.<br>(401) 1234 567<br>hello@brandi.com<br>www.brandi.com</span>
					</div>
					<div class="forma-center">
						<p>Say hello!</p>
							<div class="input">
								<input type="text" name="email" value="" placeholder="Name">
								<input type="text" name="email" value="" placeholder="Email"><br>
								<span>
									<input type="text" name="email" value="" placeholder="Massage">
								</span>
							</div>
							<a href="#"><img src="images/message.png" alt="message">Send message</a>
					</div>
					<div class="forma-social">
						<a href="#"><img src="images/discuss1.png" alt=""></a><br>
						<a href="#"><img src="images/discuss2.png" alt=""></a><br>
						<a href="#"><img src="images/discuss3.png" alt=""></a><br>
						<a href="#"><img src="images/discuss4.png" alt=""></a><br>
						<a href="#"><img src="images/discuss5.png" alt=""></a>
					</div>
			</div>
		</div>
	</div>
	<div class="karta">
		<script type="text/javascript" charset="utf-8" src="https://api-maps.yandex.ru/services/constructor/1.0/js/?sid=ZU5ke4cS4-UrUDahz3O-5frzRNAIZ_Pi&width=100%&height=215"></script>
	</div>
	<div class="line"></div>
	<div class="footer clearfix">
	<div class="container">
		<div class="footer-logo-section">
			<a class="footer-logo" href="#">Brandi</a>
			<div class="footer-text-section">
					<p>eusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
			</div>
		</div>
		<div class="subscribe">
			<p>Subscribe</p>
			<div class="check clearfix">
					<input type="text" name="email" value="" placeholder="">
					<button type="button" name="button"></button>
			</div>
			<span><p>eusmod tempor incididunt ut labore et dolore magna aliqua.</p></span>
		</div>
		<div class="explore">
			<p>Explore</p>
			<div class="menu1">
				<nav class="footer-nav">
					<ul>
						<li><a href="#">Envato</a></li><br>
						<li><a href="#">Themecurve</a></li><br>
						<li><a href="#">Kreativeshowcase</a></li><br>
						<li><a href="#">Freebiescurve</a></li>
					</ul>
				</nav>
			</div>
		<div class="menu2">
				<nav class="footer-nav">
					<ul>
						<li><a href="#">Themeforest</a></li><br>
						<li><a href="#">Microsoft</a></li><br>
						<li><a href="#">Google</a></li><br>
						<li><a href="#">Yahoo</a></li>
					</ul>
				</nav>
			</div>
		</div>
		<div class="copyrate">
			<p>Created with <img src="images/heart.png" alt="heart"> by <a href="#">themecurve</a>, exclusive for kreativeshowcase</p>
		</div>
		<div class="avtor">
			<p>© 2014 Brandi. All Rights Reserved.</p>
		</div>
		</div>
	</div>

</body>
</html>
