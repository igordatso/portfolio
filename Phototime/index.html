
<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="css/normalize.css"/>
<link rel="stylesheet" type="text/css" href="css/style.css"/>
<title>Phototime</title>
</head>
<body>

	<header>
		<div class="container">
			<nav>
				<ul>
					<li><a href="#">HOME</a></li>
					<li><a href="#">PHOTOAPP</a></li>
					<li><a href="#"><img src="images/foto1.png" alt="logo"></a></li>
					<li><a href="#">DESIGN</a></li>
					<li><a href="#">DOWNLOAD</a></li>
				</ul>
			</nav>
			<div class="header-block">
				<h1>YOUR LIFE , A PHOTO</h1>
				<p>print your life in a simple photo</p>
				<button type="button" name="button">GET STARTED</button>
			</div>
			<div class="social">
				<ul>
					<li><a href="#"><img src="images/mini3.png" alt="twiter"></a></li>
					<li><a href="#"><img src="images/mini2.png" alt="fliker"></a></li>
					<li><a href="#"><img src="images/mini1.png" alt="network"></a></li>
				</ul>
			</div>
		</div>
	</header>

		<div class="box1">
			<div class="container clearfix hend">
					<div class="text1">
						<h3>Rappresent your life with a<br> simple photo</h3>
						<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, <br>sed do eiusmod tempor incididunt ut labore et dolore <br>magna aliqua. Ut <b>enim ad minim</b> veniam, quis <br>nostrud exercitation ullamco laboris nisi <br>ut aliquip ex ea commodo consequat.</p>
						<button type="button" name="button2">GET STARTED</button>
					</div>
			</div>
		</div>

		<div class="box2">
			<div class="container clearfix">
				<div class="ifone">
					<img src="images/fone.jpg" alt="ifone"/>
				</div>
				<div class="text2">
					<h3>DESIGN</h3>
					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, <br>sed do eiusmod tempor incididunt ut labore et dolore <br>magna aliqua. Ut <b>enim ad minim</b> veniam, quis <br>nostrud exercitation ullamco laboris nisi <br>ut aliquip ex ea commodo consequat.</p>
					<button type="button" name="button2">LEARN MORE</button>
					<div class="arrow">
						<p><b>01/05</b></p>
						<img src="images/mini5.png" alt="left"/>
						<img src="images/mini6.png" alt="right"/>
					</div>
				</div>
			</div>
		</div>

		<div class="box3">
			<div class="container clearfix">
				<div class="arrow2">
					<img src="images/mini5.png" alt="left"/>
					<img src="images/mini6.png" alt="right"/>
				</div>
				<div class="items1">
					<div class="group">
						<img src="images/foto2.png" alt="foto">
						<h4>OUR COMMUNITY</h4>
						<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
					</div>
				</div>

				<div class="items2">
					<img src="images/image2.jpg" alt="image">
				</div>

				<div class="items2">
						<img src="images/image1.jpg" alt="image">
				</div>
			</div>
		</div>

		<div class="box4">
			<div class="container clearfix">
				<div class="block">
					<h2>subscribe to us community</h2>
					<div class="check">
							<input type="text" name="email" value="" placeholder="your email">
							<button type="button" name="button"><img src="images/mini4.png" alt="check"></button>
					</div>
				</div>
			</div>
		</div>

	<div class="box5">
		<div class="container clearfix">
			<h4>Download it</h4>
			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
			<div class="market">
				<div class="button1">
					<label>
					<button type="button" name="button">apple store</button>
					<div class="mini">
						<button type="button" name="button"><img src="images/mini8.png" alt="check"></button>
					</div>
					</label>
				</div>
				<div class="button2">
					<button type="button" name="button">play store</button>
					<div class="mini">
						<button type="button" name="button"><img src="images/mini7.png" alt="check"></button>
					</div>
				</div>
			</div>
		</div>
	</div>

<footer>
			<div class="container">
				<div class="links">
				<nav>
					<ul>
						<li><a href="#">credits</a></li>
						<p>|</p>
						<li><a href="#">privacy</a></li>
						<p>|</p>
						<li><a href="#">our teams</a></li>
					</ul>
				</nav>
				</div>
				<div class="logo">
					<img src="images/foto2.png" alt="logo">
				</div>
				<div class="copyright">
					<p>Copyright by phototime - all right reserved</p>
				</div>
		</div>
</footer>
</body>
</html>
