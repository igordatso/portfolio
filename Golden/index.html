
<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Golden</title>
	<link rel="stylesheet" type="text/css" href="css/normalize.css"/>
	<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
	<link rel="stylesheet" type="text/css" href="css/bootstrap-theme.min.css">
	<link rel="stylesheet" type="text/css" href="css/style.css"/>
</head>
<body>

<header>
	<div class="container">
		<div class="row">

				<nav class="navbar">
		  		<div class="container">
						<div class="navbar-header">
								<a class="navbar-brand" href="#">
									<img src="images/logo.png" alt="logo">
								</a>
								<button type="button" class="navbar-toggle glyphicon glyphicon-align-justify" data-toggle="collapse" data-target="#responsive-menu"></button>
						</div>

						<div class="text-center collapse navbar-collapse" id="responsive-menu">
							<ul class="nav navbar-nav">
								<li><a href="#">Home</a></li>
								<li><a href="#">Services</a></li>
								<li><a href="#">Portfolio</a></li>
								<li><a href="#">About</a></li>
								<li><a href="#">Contact</a></li>
							</ul>
						</div>

					</div>
				</nav>

				<span>Welcome To Our Studio!</span>
				<h1>Itâ€™s nice to Meet you</h1>
				<button type="button">Tell Me More</button>

		</div>
	</div>
</header>

	<div class="services">
		<div class="container">
			<div class="row">
				<h2>Services</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
				<div class="col-md-4 block">
					<img src="images/services1.png" alt="services1">
					<p>E-Commerce</p>
					<span>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</span>
				</div>
				<div class="col-md-4 block">
					<img src="images/services2.png" alt="services2">
					<p>Responsive Web</p>
					<span>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</span>
				</div>
				<div class="col-md-4 block">
					<img src="images/services3.png" alt="services3">
					<p>Web Security</p>
					<span>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</span>
				</div>
			</div>
		</div>
	</div>

	<div class="portfolio">
		<div class="container ramka">
			<h2>Our portfolio</h2>
			<p>Proin iaculis purus consequat sem cure.</p>
			<div class="row">
				<div class="col-md-4 col-sm-6 col-xs-12 block">
					<div class="serv-block">
						<img src="images/portfolio1.jpg" alt="portfolio1" class="img-responsive">
						<p>Ebony & Ivory</p>
						<span>Branding</span>
					</div>
				</div>
				<div class="col-md-4 col-sm-6 col-xs-12 block">
					<div class="serv-block">
						<img src="images/portfolio2.jpg" alt="portfolio2" class="img-responsive">
						<p>Smart Stationary</p>
						<span>Print Design</span>
					</div>
				</div>
				<div class="col-md-4 col-sm-6 col-xs-12 block">
					<div class="serv-block">
						<img src="images/portfolio3.jpg" alt="portfolio3" class="img-responsive">
						<p>Smart Stationary</p>
						<span>Print Design</span>
					</div>
				</div>
				<div class="col-md-4 col-sm-6 col-xs-12 block">
					<div class="serv-block">
						<img src="images/portfolio4.jpg" alt="portfolio4" class="img-responsive">
						<p>Vinyl Record</p>
						<span>Product Mock-Up</span>
					</div>
				</div>
				<div class="col-md-4 col-sm-6 col-xs-12 block">
					<div class="serv-block">
						<img src="images/portfolio5.jpg" alt="portfolio5" class="img-responsive">
						<p>Treehouse Template</p>
						<span>Web Design</span>
					</div>
				</div>
				<div class="col-md-4 col-sm-6 col-xs-12 block">
					<div class="serv-block">
						<img src="images/portfolio6.jpg" alt="portfolio6" class="img-responsive">
						<p>Burned Logo</p>
						<span>Branding</span>
					</div>
				</div>
			</div>
		</div>
	</div>

	<div class="about_us">
		<div class="container">
			<div class="row clearfix">
				<h2>About Us</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
			</div>

				<div class="row line-right">
				<div class="col-md-4 visible-sm visible-xs">
					<img src="images/about1.png" alt="about1">
				</div>
				<div class="col-md-4">
					<div class="date">
						<p>July 2010</p>
					</div>
					<p>Our Humble Beginnings</p>
					<span>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</span>
				</div>
				<div class="col-md-4 visible-md visible-lg">
					<img src="images/about1.png" alt="about1">
				</div>
				<div class="col-md-4">
				</div>
				</div>

				<div class="row line-left">
				<div class="col-md-4">
				</div>
				<div class="col-md-4">
					<img src="images/about2.png" alt="about2">
				</div>
				<div class="col-md-4">
					<div class="date">
						<p>January 2011</p>
					</div>
					<p>Facing Startup Battles</p>
					<span>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</span>
				</div>
				</div>

				<div class="row line-right">
				<div class="col-md-4 visible-sm visible-xs">
					<img src="images/about1.png" alt="about1">
				</div>
				<div class="col-md-4">
					<div class="date">
						<p>December 2012</p>
					</div>
					<p>Enter The Dark Days</p>
					<span>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</span>
				</div>
				<div class="col-md-4 visible-md visible-lg">
					<img src="images/about3.png" alt="about3">
				</div>
				<div class="col-md-4">
				</div>
				</div>

				<div class="row line-left">
				<div class="col-md-4">
				</div>
				<div class="col-md-4">
					<img src="images/about4.png" alt="about4">
				</div>
				<div class="col-md-4">
					<div class="date">
						<p>february 2014</p>
					</div>
					<p>Our Triumph</p>
					<span>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.</span>
				</div>
				</div>

				<div class="row center">
				<div class="col-md-4">
				</div>
				<div class="col-md-4">
					<img src="images/about5.png" alt="about4">
				</div>
				</div>



		</div>
	</div>

	<div class="amazing">
		<div class="container">
			<div class="row clearfix">
				<h2>our amazing team</h2>
				<p>Proin iaculis purus consequat sem cure. </p>

					<div class="col-md-4 block">
						<img src="images/amazing_elipse.png" alt="elipse1">
						<div class="text">
							<p>Kimberly Thompson</p>
							<span>Marketer</span>
						</div>
						<div class="social1"></div>
						<div class="social2"></div>
						<div class="social3"></div>
					</div>
					<div class="col-md-4 block">
						<img src="images/amazing_elipse.png" alt="elipse2">
						<div class="text">
							<p>Rico Massimo</p>
							<span>Coder</span>
						</div>
						<div class="social1"></div>
						<div class="social2"></div>
						<div class="social4"></div>
					</div>
					<div class="col-md-4 block">
						<img src="images/amazing_elipse.png" alt="elipse3">
						<div class="text">
							<p>Uku Mason</p>
							<span>Graphic Designer</span>
						</div>
						<div class="social1"></div>
						<div class="social3"></div>
					</div>
					<div class="amasing_text">
						<p>Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
					</div>
			</div>
		</div>
	</div>

	<div class="sponsors">
		<img src="images/amazing1.jpg" alt="amazing1">
		<img src="images/amazing2.jpg" alt="amazing2">
		<img src="images/amazing3.jpg" alt="amazing3">
		<img src="images/amazing4.jpg" alt="amazing4">
	</div>

	<div class="contact">
		<div class="container">
			<div class="row clearfix">
				<h2>Contact us</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
					<div class="col-md-6">
						<div class="form-group forma">
							<input type="text" class="form-control" placeholder="your Name *">
						</div>
						<div class="form-group forma">
							<input type="text" class="form-control" placeholder="your e-mail *">
						</div>
						<div class="form-group forma">
							<input type="text" class="form-control" placeholder="subject">
						</div>
					</div>
					<div class="col-md-6 forma">
						<textarea class="form-control" placeholder="Your message *"></textarea>
					</div>
				</div>
				<div class="row clearfix">
					<button type="button">Send Message</button>
				</div>
			</div>
		</div>

	 	<div class="footer">
			<div class="container">
				<div class="row clearfix">
					<div class="col-md-12 col-xs-12 social">
						<div class="social1"></div>
						<div class="social2"></div>
						<div class="social3"></div>
						<div class="social4"></div>
					</div>
				</div>
			</div>
	 	</div>

	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
	<script src="js/bootstrap.min.js" type="text/javascript"> </script>
</body>
</html>
